recipes-support/provisioning/files/phytec-board-config.sh

```diff
@@ -57,8 +60,8 @@ calc_wt_size() {
 
 do_about() {
 	whiptail --msgbox "\
-This tool provides a straightforward way of doing initial
-configuration of your PHYTEC board for the
+This tool (version ${version})  provides a straightforward
```

Comment:
I do not see the need to add the version number here. If it is needed one can just call the program with the dedicated option you added. Maybe add the version to the `--backtitle` of the main setup window, alongside the device tree model name.

Reply:
i remove the version at this point